lib/main.js

```diff
@@ -40,11 +41,22 @@ function shim(Linter) {
   return LinterShim;
 }
 
+/**
+ * Checks whether or not the given grammar is for TypeScript
+ * @param {Grammar} grammar The grammar to check
+ * @return {boolean} Whether or not the grammar is for TypeScript
+ */
+function isTypescriptGrammar(grammar) {
+  return grammar.fileTypes.find(fileType => /.*ts/i.test(fileType));
```

Comment:
Anything that isn't listed in the [Atom Docs](https://atom.io/docs/api/v1.15.0/AtomEnvironment) isn't part of the public API, and is subject to change at any time 😉.

In this case the [Grammar](https://atom.io/docs/api/v1.15.0/Grammar) object you are accessing is _very_ bare bones, and doesn't expose a way to get to the `fileTypes` property you are utilizing here. [This comment](https://github.com/AtomLinter/linter-tslint/pull/149#discussion_r108806207) shows a way around all this 😉.